internal/cli/terraform_fetcher.go

```diff
@@ -237,6 +240,41 @@ func (f *pagesResourceFetcher) FetchData(_ context.Context) (importDataList, err
 	}, nil
 }
 
+func (f *resourceServerResourceFetcher) FetchData(ctx context.Context) (importDataList, error) {
+	var data importDataList
+
+	var page int
+	for {
+		apis, err := f.api.ResourceServer.List(
+			ctx,
+			management.Page(page),
```

Comment:
We could add as well the management.PerPage(100) just to limit a bit the amount of potential requests. 🤔